Riot/Modules/Spaces/SpaceList/SpaceListViewModel.swift

```diff
@@ -38,10 +38,14 @@ final class SpaceListViewModel: SpaceListViewModelType {
     
     init(session: MXSession) {
         self.session = session
+        
+        NotificationCenter.default.addObserver(self, selector: #selector(self.sessionDidSync(notification:)), name: MXSpaceService.didBuildSpaceGraph, object: nil)
+
     }
     
     deinit {
         self.cancelOperations()
+        NotificationCenter.default.removeObserver(self)
```

Comment:
Not necessary to remove the observer in the deinit when you register with `addObserver(_:selector:name:object:)` as said here: https://developer.apple.com/documentation/foundation/notificationcenter/1415360-addobserver 
> If your app targets iOS 9.0 and later or macOS 10.11 and later, you do not need to unregister an observer that you created with this function. If you forget or are unable to remove an observer, the system cleans up the next time it would have posted to it.